src/Saas.Admin/Saas.Admin.Service/Program.cs

```diff
@@ -121,8 +117,6 @@
     })
     .ConfigureHttpClient(options =>
     {
-        options.BaseAddress = new Uri(builder.Configuration["PermissionsApi:BaseUrl"]);
```

Comment:
this is still needed. This is telling the admin service client where to hit the permissions api at